commands/inventory/add_agent_mongodb_exporter.go

```diff
@@ -101,4 +103,5 @@ func init() {
 	AddAgentMongodbExporterC.Flag("skip-connection-check", "Skip connection check").BoolVar(&AddAgentMongodbExporter.SkipConnectionCheck)
 	AddAgentMongodbExporterC.Flag("tls", "Use TLS to connect to the database").BoolVar(&AddAgentMongodbExporter.TLS)
 	AddAgentMongodbExporterC.Flag("tls-skip-verify", "Skip TLS certificates validation").BoolVar(&AddAgentMongodbExporter.TLSSkipVerify)
+	AddAgentMongodbExporterC.Flag("use_vmagent", "When false, pull model is used. By default, its true").Default("true").BoolVar(&AddAgentMongodbExporter.UseVMAgent)
```

Comment:
Have you discussed about usage of this flag?

Reply:
@AlekSi  suggested to add `use_vmagent` to inventory api, with it user can disable push model for some exporters. 

By default, push model detects at the server by matching client version for management api and by reading this flag for inventory api.

Reply:
does this mean that vm-agent doesn't support pull model?

Reply:
old code